### DISCUSSION TITLE
Best way to share info with follower

### DISCUSSION THREAD

**Discussion by nickrout** [Navid200/xDrip#18] [General]
I have dabbled with xDrip in the past, but now I am using a Dexcom G6 it is coming in to its own, and it is so much better than the dexcom crap app.

So far I have got G6-->xDrip+ -->NightScout (also -->Home Assistant, but that isn't relevant to my question).

I want to share my GSM data with my wife. I intend to install xDrip+ on her phone. I watched this video https://www.youtube.com/watch?v=LcgjfbYcWkE but it is very old (2016) and I wasn't clear at the end how my xDrip+ communicates with her xDrip+. Does one require a parakeet?

Is there another/better way? I see a menu item for Hardware Data Source|Nightscout follower. Am I better just to set her up as a Nightscout follower?

I love Nightscout and xDrip+, its just that there are so many options.

Thanks in advance, Nick.

**Comment by Navid200** [comment]
If you already have Nightscout set up, you can use Nightscout follower on the follower phone.  You only need to enter the web address as the url, not the password.  So, you only need to enter what you enter in a web browser to access your readings on the follower phone.

The other option, which you have found the clip for it, is the xDrip follower.
There are pros and cons for those.  The xDrip sync uses Google messaging.  There are reports of a sync error coming up due to issues with Google.  So, I always recommend Nightscout to begin with.

**Comment by nickrout** [comment]
Thank you so much.

**Reply by Navid200** [reply]
You're welcome
